go.mod

```diff
@@ -24,16 +26,22 @@ require (
 	github.com/spf13/viper v1.18.2
 	golang.org/x/crypto v0.19.0
 	golang.org/x/net v0.21.0
-	k8s.io/api v0.28.4
+	k8s.io/api v0.29.0-rc.1
```

Comment:
I would not use a RC version here

Reply:
That's the `tidy` command work here, should I just move it to `v0.29.0` ?

Reply:
Done